shell/browser/ui/message_box_gtk.cc

```diff
@@ -33,6 +31,27 @@ MessageBoxSettings::~MessageBoxSettings() = default;
 
 namespace {
 
+// Copied from L40-L55 in
+// https://cs.chromium.org/chromium/src/chrome/browser/ui/libgtkui/select_file_dialog_impl_gtk.cc
```

Comment:
i don't love that we straight up copy this exact big block twice...can we somehow reuse this across both instances perhaps? 🤔

Reply:
Good suggestion! Fixed 4ac3f14258becac3ed2f1bf07e659755ad24c504.